PauloAlves8039/dotnet-parking-backend
# [FEATURE] Created Controllers and Endpoints

### Context
This Pull Request implements the `Controllers` and `Endpoints` with their mappings coming from the domain classes to the DTOs classes.

### Changes
**Controllers**: Created Endpoints for CRUD:

- `CustomersController`
- `VehicleController`
- `CustomerVehicleController`
- `AddressController`
- `StayController`


### How to Test
Check the call for services on the Controllers.
Ensures that CRUD Endpoints work as expected.

### Notes
- Review Controllers implementations.
- Thoroughly test all all Endpoints and extreme cases.

### Related Issues
Link to related issues or tasks, if applicable.
